keras_cv/layers/object_detection/anchor_generator_test.py

```diff
@@ -189,6 +189,8 @@ def test_relative_generation(self):
             clip_boxes=False,
         )
         boxes = anchor_generator(image=image)
-        boxes = np.concatenate(list(boxes.values()), axis=0)
+        boxes = np.concatenate(
```

Comment:
`ops.concatenate` here, or `np.array` below? otherwise you are converting to a backend tensor and then back to np in the same line.

Reply:
Discussed offline -- we're not converting to a backend tensor here. All numpy all the way 😀